pkg/apis/core/v1alpha1/zz_generated.kubebuilder.go

```diff
@@ -413,7 +413,6 @@ var (
 			Scope: "Namespaced",
 			Validation: &v1beta1.CustomResourceValidation{
 				OpenAPIV3Schema: &v1beta1.JSONSchemaProps{
-					Type: "object",
```

Comment:
Did some changes sneak in unintentionally here? Do we need a new verify task to ensure this doesn't happen again?

Reply:
This is auto-generated. the code which we have in master probably is generated with older version of kubebuilder.